### DISCUSSION TITLE
Daily 03/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#197] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy?
- Dificultades e impedimentos

**Comment by rpgrca** [comment]
Hoy aporté un diseño simple para el ABM de catálogo, si hay más tareas de diseño las voy a ir haciendo así, de puño y letra. También me gustaría bajarme todo el dev completo para ver cómo está el sitio, limpiar un poco el código y podemos empezar a limpiar un poco el estilo ya que parece que llegamos con 2 días de anticipación a la fecha original.

**Comment by BraianTroncoso** [comment]
Ayer terminé el componente carrito del path "Mis compras" con todas sus funciones principales, también le agregé productos para poder hacer el testing completo.
Hoy voy a terminar lo de Ingles, también tengo clase de Java en la UTN, asi que voy a ver si puedo refactorizar algo de código o solamente leer mas documentacion.
Tengo que seguir con el curso de Angular también.

**Comment by Brian-Almada** [comment]
Ayer viendo Angular. Hoy también.

**Comment by NataliaAlvarezIspc** [comment]
Hoy estuve terminando la página de modificación de datos para la registración. la terminé y la subí al repositorio. También estuve viendo como mejorar las validaciones en los formularios, cosa que haré mañana
